### PR TITLE
Run Codacy reporter from Travis CI

### PR DESCRIPTION
**Pull request by KarolJastrzebski**
Adds code coverage reporting in Codacy.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by rkondratowicz** [comment]
Do we not also need a `jacocoTestReport` section in build.gradle?

**Comment by rkondratowicz** [comment]
Could you also add a badge to readme?

File: .travis.yml [hunk 1]
```diff
@@ -9,6 +9,8 @@ before_cache:
 
 before_install:
   - curl https://raw.githubusercontent.com/hmcts/reform-api-docs/master/bin/publish-swagger-docs.sh > publish-swagger-docs.sh
+  - sudo apt-get install jq
```
Comment (rkondratowicz) [inline .travis.yml:12]: does `sudo: required` not need to be declared?

Comment (KarolJastrzebski) [reply .travis.yml:12]: It seems not required as builds are passing and data are reported correctly.

**Review by rkondratowicz — APPROVED** [review]

**Pull request merged**